client/app/telecom/telephony/alias/configuration/feature/conference/telecom-telephony-alias-configuration-conference.html

```diff
@@ -0,0 +1,179 @@
+<oui-spinner data-ng-if="$ctrl.loading"></oui-spinner>
+
+<div data-ng-if="!$ctrl.loading">
+    <form novalidate
+          name="conferenceConfiguration">
+        <!-- CONFIGURATION -->
+        <h3 class="oui-heading_underline"
+            data-translate="telephony_alias_config_title"></h3>
+
+        <oui-field data-label="{{:: 'telephony_alias_config_conference_access_pin_code' | translate }}"
+                   data-help-text="{{:: 'telephony_alias_config_conference_access_pin_code_help' | translate }}"
+                   data-size="s">
+            <input class="oui-input"
+                   type="text"
+                   id="pincode"
+                   name="pincode"
+                   data-ng-model="$ctrl.conference.pin"
+                   data-ng-pattern="/^0|([0-9]{8})$/">
+        </oui-field>
+        <dl>
+            <dt class="oui-label">
+                <span data-translate="telephony_alias_config_conference_customized_announcement"></span>
+                <oui-popover>
+                    <button class="oui-popover-button"
+                            type="button"
+                            data-oui-popover-trigger></button>
+                    <oui-popover-content>
+                        <span data-translate="telephony_alias_config_conference_customized_announcement_help"></span>
+                    </oui-popover-content>
+                </oui-popover>
+            </dt>
+            <dd data-ng-if="!$ctrl.newFile"
+                data-ng-bind="$ctrl.conference.announceFile ? $ctrl.conference.announceFilename : ('common_none' | translate)"></dd>
+            <dd>
+                <tuc-input-file data-ng-model="$ctrl.newFile"
+                                data-ng-accept="audio/*"
+                                data-ng-accept-filter="$ctrl.checkValidAudioExtention(file)">
+                    <span class="fa fa-upload mr-1"></span>
```

Comment:
I suppose there is no `oui-icon` equivalent ?

Reply:
No there is not

Reply:
```suggestion
                    <span class="fa fa-upload mr-1" aria-hidden="true"></span>
```